README.md

```diff
@@ -74,6 +74,10 @@ The required version of Xcode changes frequently, and is often a beta release.
 Check this document or the host information on <https://ci.swift.org> for the
 current required version.
 
+Be aware that the Command Line Tools version must match the required Xcode version. 
```

Comment:
One last thing:

> Be aware that the Command Line Tools version must match the required Xcode version. Command Line Tools can be managed using the xcode-select tool and, in Xcode, at the Xcode>Preferences>Locations tab.

Let's try to merge this with the paragraph above:

To build for macOS, you need Xcode 10 beta. The required version of Xcode changes frequently, and is often a beta release. Be aware that the Command Line Tools version must match the required Xcode version. Command Line Tools can be managed using `xcode-select` or the Locations tab in Xcode Preferences.  Check this document or the host information on https://ci.swift.org for the current required version.